ourjapanlife/findadoc-server
# build: install Graphql, Typescript, and Apollo Server

# Part of Issue #2 

1. Installs GraphQL, TypeScript, and Apollo Server
2. Creates a basic schema with 1 query
3. Creates a server at `localhost:3000`

## How to Test

1.  Run `npm run dev`
2. Open your browser to `http://localhost:3000/`
3. Navigate to the **Explorer** section from the menu in the left pane.
4. Run the following query and confirm that you get a response with status code 200:

```
query Query {
  ok
}
```

![Screen Shot 2022-06-11 at 7 24 39 PM](https://user-images.githubusercontent.com/31802656/173184087-2957b5a0-fca3-461f-94be-66936b997c95.png)


## TODO
The README and LICENSE will be updated in a separate PR. Since this repo is private, we should decide if we want to use the same BSD license that we use in the [findadoc-web repo](https://github.com/ourjapanlife/findadoc-web).
